executor/aggfuncs/aggfunc_test.go

```diff
@@ -263,6 +263,55 @@ func rowMemDeltaGens(srcChk *chunk.Chunk, dataType *types.FieldType) (memDeltas
 	return memDeltas, nil
 }
 
+type multiArgsUpdateMemDeltaGens func(*chunk.Chunk, []*types.FieldType) (memDeltas []int64, err error)
+
+func defaultMultiArgsMemDeltaGens(srcChk *chunk.Chunk, dataTypes []*types.FieldType) (memDeltas []int64, err error) {
+	memDeltas = make([]int64, 0)
+	m := make(map[string]bool)
+	for i := 0; i < srcChk.NumRows(); i++ {
+		row := srcChk.GetRow(i)
+		if row.IsNull(0) {
+			memDeltas = append(memDeltas, int64(0))
+			continue
+		}
+		datum := row.GetDatum(0, dataTypes[0])
+		if datum.IsNull() {
+			memDeltas = append(memDeltas, int64(0))
+			continue
+		}
+
+		memDelta := int64(0)
+		key, err := (&datum).ToString()
```

Comment:
Why we need `(&datum)` here?

Reply:
we use datum to check whether it is null, and then convert to string

Reply:
can we use  `datum.ToString()`  here? @TszKitLo40

Reply:
Yes, I will fix it.